users/serializers/common.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = ('username', 'first_name', 'last_name', 'email', 'password',)
+        extra_kwargs = {'password': {'write_only': True}}
+
+    def create(self, validated_data):
+        User = get_user_model()
+
+        user = User(
```

Comment:
Its actually fine to throw a `KeyError`. Probably want to log it, but will save that for later.